rock_gazeboTypes.hpp

```diff
@@ -9,8 +9,12 @@
 namespace rock_gazebo
 {
     struct LinkExport {
-        // The port name
+        // The RBS port name
         std::string port_name;
+        // The RBA port name
+        std::string rba_port_name;
```

Comment:
Instead of allowing any name for acceleration and wrench, I would prefer having a naming convention that creates known wrench / acceleration ports based on the port_name, e.g.

```
port_name = 'rotor';
acceleration => "rotor_accelerations"
wrench => "rotor_wrench"
```